src/view/window/PreviewWindow.js

```diff
@@ -32,24 +32,38 @@ Ext.define('BasiGX.view.window.PreviewWindow', {
         dataExtent: [[10000, 20000], [20000, 40000]]
     },
 
-    tools: [{
-        type: 'gear',
-        handler: function() {
-            this.up('window').showLegend();
-        }
-    }],
-
     /**
      * Sets up the map and the controls.
      */
     initComponent: function() {
         this.callParent();
         var markup = '<i class="fa fa-globe" aria-hidden="true"></i>';
         var worldIcon = Ext.dom.Helper.createDom(markup);
+
+        /**
+        * Create custom button to show legend.
+        */
+        var buttonInfo = document.createElement('button');
+        buttonInfo.title = 'Show legend';
+        buttonInfo.type= 'button';
+        buttonInfo.innerHTML = '<i class="fa fa-info" aria-hidden="true"></i>';
+
+        var showLegendWindow = function() {
+            this.showLegend();
+        };
+        buttonInfo.addEventListener('click', showLegendWindow.bind(this));
```

Comment:
Cant we just call and bind `this.showLegend` here?

Reply:
Yes, I think so (untested)

 buttonInfo.addEventListener('click', showLegendWindow.bind(this));

```suggestion
        buttonInfo.addEventListener('click', this.showLegend.bind(this));
```